app/controllers/usercontroller.js

```diff
@@ -92,8 +88,20 @@ class UserController {
       res.status(400).send({ error: e.message });
     }
   }
+  async login(req, res) {
+
+    try {
+      const { email, password } = req.body;
+      const user = await this.getEmail(email);
+      await this.validatePassword(password, user.password);
+      res.status(201).send({ message: "sucesso" });
+
+    } catch (e) {
+      res.status(500).send({ error: e.message });
+    }
+  }
 
-  async validatePassword(password, passwordToCompare) {    
+  async validatePassword(password, passwordToCompare) {
```

Comment:
Seria interessante passar a mensagem de erro via parâmetro, caso não seja passado nada ele retorna a mensagem que já existe. Se for passado uma mensagem ele envia a mensagem passada.